Assignment_3_Piano_2/app/src/main/res/layout/fragment_full_tone_piano_key.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    tools:context=".FullTonePianoKeyFragment">
+
+    <Button
+        android:id="@+id/fullToneKey"
+        android:layout_width="60dp"
+        android:layout_height="355dp"
```

Comment:
Istedet for height="355dp" så kan du bruke Height_percent="0.6" for eksempel, kanskje lettere neste gang :)